--- a/src/Servant/GitHub/Webhook.hs	2026-07-13 17:56:16
+++ b/src/Servant/GitHub/Webhook.hs	2026-07-13 17:56:16
@@ -92,6 +92,7 @@
 import Crypto.MAC.HMAC ( hmac, HMAC(..) )
 import Data.Aeson ( decode', encode, Value(String, Object) )
 import qualified Data.Aeson as Aeson
+import qualified Data.Aeson.KeyMap as KeyMap
 import qualified Data.Aeson.Types as AesonType
 import Data.ByteArray ( convert, constEq )
 import qualified Data.Text as T
@@ -225,8 +226,8 @@
 
 instance HasRepository AesonType.Object where
     getFullName o =
-        do Object r <- HashMap.lookup "repository" o
-           String n <- HashMap.lookup "full_name" r
+        do Object r <- KeyMap.lookup "repository" o
+           String n <- KeyMap.lookup "full_name" r
            pure n
 
 -- |For use with 'github-webhooks' package types.  Routes would look like:
